### PR TITLE
add printics feature as mentioned in issue #469

### PR DESCRIPTION
**Pull request by dopefishh**

### CODE REVIEW

**Comment by hobarrera** [comment]
LGTM. I think we need tests for this as well though.

**Comment by untitaker** [comment]
Yes please, the new function has no test coverage. You can add a test in either https://github.com/pimutils/khal/blob/master/tests/controller_test.py or https://github.com/pimutils/khal/blob/master/tests/cli_test.py

**Comment by geier** [comment]
feel free to ask for help if it isn't clear how to write some tests for this. Good work btw!

**Comment by dopefishh** [comment]
I'll try to do this today or tomorrow. If there are any questions I'll let you guys know.

**Comment by dopefishh** [comment]
I've added some tests (sorry it took so long). But it should be functional and tested now. While testing I've noticed that I had to change the config spec as well:)

**Comment by dopefishh** [comment]
something went wrong with merging back into master. I'll fix this

**Comment by dopefishh** [comment]
@geier all test succeed:)

**Comment by geier** [comment]
Sorry for not having had a propper look at this. I'll try ASAP.

**Comment by dopefishh** [comment]
NP, I was very late as well

File: khal/controllers.py [hunk 1]
```diff
@@ -585,3 +586,25 @@ def import_event(vevent, collection, locale, batch, format=None, env=None):
...
+    if format is None:
+        format = conf['view']['agenda_event_format']
+    cal = icalendar.Calendar.from_ical(ics)
+    events = [item for item in cal.walk() if item.name == 'VEVENT']
+    events_grouped = defaultdict(list)
+    for event in events:
+        events_grouped[event['UID']].append(event)
+
+    if format is None:
+        format = conf['view']['event_format']
```
Comment (geier) [inline khal/controllers.py]: I believe this second `if format is None` is superfluous, see line 591. Also according to the text in khal.spec `event_format` is the right format string, the current default just sucks.

**Comment by geier** [comment]
one more note: please also add a new paragraph in `doc/source/usage.rst`. Thank you!

**Comment by dopefishh** [comment]
I think I've squashed it correctly and reviewed your changes:)

**Comment by geier** [comment]
οne more thing (sorry!): please run `make html` in the `doc` directory, that should produce a diffing configspec.rst file, and also check that in (you might need to install sphinxcontrib-newsfeed).

**Comment by dopefishh** [comment]
np. Done

**Comment by geier** [comment]
I'm sorry to  bother you again, but are the changes to the configwizard intentionally in this commit? They look suspiciously like my changes to the configwizard I commited recently, perhaps a merge/rebase gone wrong?

**Comment by dopefishh** [comment]
Probably not... My git-fu is not that great so it seems...

**Comment by geier** [comment]
an alternative might be to create a new branch (off master) and copying
over the relevant files and creating a new PR.

Quoting dopefishh (2017-01-10 11:00:45)

**Comment by dopefishh** [comment]
at last:) this should do it

**Comment by geier** [comment]
Thank you a lot and merged!

**Pull request merged**